### PR TITLE
💄 Add navigation bar and wrapper to frontend 🧭

### PR DESCRIPTION
**Pull request by lisamittmann**
- add navigation wrapper 
- add navigation bar
- include new components in `app.js`
- adjust recipe overview to allow for navigation bar

### CODE REVIEW

File: frontend/src/components/NavigationComponents/NavigationBar.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <NavigationWrapper>
+            <NavigationLink to="/recipes">REZEPT ÜBERSICHT</NavigationLink>
+            <NavigationLink to="/">CORONA LAGE</NavigationLink>
+            <NavigationLink to="/">MEIN BEREICH</NavigationLink>
+        </NavigationWrapper>
+
+    )
+}
+
+const NavigationWrapper = styled.section`
```
Comment (fabianschmauder) [inline frontend/src/components/NavigationComponents/NavigationBar.js]: it is a little bit confusing that you have NavigationWrapper here and another component with the same name in the same directory

File: frontend/src/App.js [hunk 2]
```diff
@@ -2,24 +2,29 @@ import './App.css';
...
-                    <p>This is my starter page</p>
-                    <Link to="/recipes">Rezept-Übersicht</Link>
-                </Route>
-                <Route path="/recipes">
-                    <RecipeOverview/>
-                </Route>
                 <Route path="/recipe/:recipeId">
                     <RecipePage/>
                 </Route>
+                <NavigationWrapper>
```
Comment (fabianschmauder) [inline frontend/src/App.js]: Maybe there is a better name for NavigationWrapper like Page Layout?

**Review by fabianschmauder — COMMENTED** [review]
LGTM 😉

File: frontend/src/components/NavigationComponents/NavigationBar.js (hunk 1 above)
Comment (lisamittmann) [reply frontend/src/components/NavigationComponents/NavigationBar.js]: very true, will change that

**Pull request merged**